### DISCUSSION TITLE
Testimonials for EddieHub website

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#1618] [Show and tell]
We would like to include your testimonials for the new **EddieHub** website.

- Please write 1 sentence below about what being part of the **EddieHub** community means to you?
- Include a star rating out of 5

Example...

<img width="1171" alt="Screenshot 2021-05-11 at 12 17 24" src="https://user-images.githubusercontent.com/624760/117806937-df660680-b252-11eb-8b17-c95b96aa4096.png">

**Comment by ghost** [comment]
EddieHub is like a family to me! We get really warm welcomes here! We get all our issues solved by each others' #help :) We get our 🟩 treats here! EddieHub is awesome. By Awesome I mean Awesome! It is a 10/5 rating!

**Reply by eddiejaoude** [reply]
wow thank you 💥

**Comment by starlightknown** [comment]
It's everything anyone could ever ask for in a community. Wholesome, Encouraging, Open Source and everything nice! Best thing happened in 2020. I'll be always thankful to this community and keep contributing here. It's beyond rating of 5 stars. 3000/5 ⭐

**Reply by eddiejaoude** [reply]
amazing! thank you 💥

**Comment by DevCassie** [comment]
EddiehubCommunity for me was the start of open source development 🟩 . As a designer, I've come across many software developers within the community that is always very helpful and enthusiastic about that green square. I didn't realize why it was so important at first. But I can finally say why. In the community, they don't just help each other when they are stuck with something. They collaborate on projects, sharing knowledge and experience with each other every day, they help to build each other's future so to speak. Which is amazing to see. Not only do they geek out 🤓 , but they also show support for people in their exams. This is why I enjoy spending time in the community.

**Reply by eddiejaoude** [reply]
Great to hear, thank you 💥

**Comment by kaiwalyakoparkar** [comment]
This is just an awesome community. This is the first community I have seen which puts collaboration first than the code. Collaborative people make a collaborative community and EddieHub is a perfect example of it :smile:. I can't rate this community on the scale because priceless things can't be quantified :heart:

**Reply by eddiejaoude** [reply]
> I can't rate this community on the scale because priceless things can't be quantified ❤️

oh wow!! 💥

**Reply by kaiwalyakoparkar** [reply]
Thank you very much

**Comment by kaiwalyakoparkar** [comment]
This community has almost everything. You make great friends who share the same goal and vision. Everyone here collaborates here as a single-family and in a manner that no one feels left out. If you have any problem and suggestion community is always open to hear you. If you speak there are minium 100 ears listening to you and correct you at every point and help you grow :heart:

**Reply by eddiejaoude** [reply]
thank you!!

**Comment by kaiwalyakoparkar** [comment]
**Before joining EddieHub:** Self-doubt, lack of motivation, no fixed goals, confusion, stuck, searching for guidance
**After joining EddieHub:** What are these things? Do I even know them? (In context to - Self-doubt, lack of motivation, no fixed goals, confusion, stuck, searching for guidance)

**Reply by eddiejaoude** [reply]
Before and after, amazing idea 💥

**Reply by kaiwalyakoparkar** [reply]
I think it can be added as a meme :wink:

**Comment by kaiwalyakoparkar** [comment]
EddieHub is like a perfect portion of coffee powder, sugar & milk in a cup. Consists of everything yet perfect :heart:

**Reply by eddiejaoude** [reply]
lol 💥

**Comment by kaiwalyakoparkar** [comment]
EddieHub is the place where a where everyone has a different background, different knowledge, and expertise level yet share a single vision and goal i.e Open source is for everyone and Collaboration 1st Code 2nd

**Reply by eddiejaoude** [reply]
so many great testimonials !!! 💥

**Comment by vinzvinci** [comment]
Joining on #EddieHub is one of the greatest achievements that you can flex to everyone around you, always love this community for being supportive to many people, plus being inclusive and diverse (5/5).

**Reply by eddiejaoude** [reply]
amazing! thank you 💥

**Comment by Vyvy-vi** [comment]
EddieHub is a great place to catalyse growth in both personal and development spheres. The community really helps you get accustomed to social coding, which is an essential skill. Contributing with EddieHub has taught me how to write better issues and how to better explain things, by giving enough context.

The community is highly diverse and consists of developers ranging from experts to people who just started programming, giving you an ideal place to ask for help, reviews and ideas. Almost everyone in the community is ready to jump in and help.

The community also inculcates better speech tone and wiser use of words, due to the presence of Alex.js on the discord bot, which really trains you to communicate better and use more inclusive vocabulary.

**Reply by eddiejaoude** [reply]
great! thank you 💥

**Comment by Dhriti-Roy** [comment]
EddieHub is a place where we all connect, explore and experience awesome team work....Each day we grow, we make new connections and above all we support each other. EddieHub is a family, we help each other in every stage. It's a complete package of everything you want in a community. it's ideal place to start your Open-Source journey. And, even if you don't know how to code...you are still welcomed...because EddieHub is all about community, we start each day with the awesome quote - Community first, Code second.

**Reply by eddiejaoude** [reply]
wow thank you!! 💥

**Comment by jaspermayone** [comment]
I discovered EddieHub fairly recently, but am really loving it! It a thoughtful, kind, welcoming place with an amazing team and community behind it! You can feel the saying/motto "Community first, Code second" being embodied by all their staff and community. I would give it 5/5 stars and would give more if I could!

**Reply by jaspermayone** [reply]
How does that look @eddiejaoude?

**Reply by eddiejaoude** [reply]
that is super great, thank you! 💥

**Reply by jaspermayone** [reply]
Of course!

**Comment by shikharcodes** [comment]
EddieHub is a place where we interact, help, learn and grow along with folks from around the world, from experts to beginners having diverse backgrounds and cultures, which makes it even more interesting and fun. It is a very inclusive community where every member is eager to assist each other, and the best part is that one gets to learn new things daily just through good conversations. 

It's delightful working along with fellow member friends sharing similar goals and vision. Contributing with the community has helped me to develop in myself the healthy practices of doing open source contributions, maintaining good documentation for projects, contributing and working in collaboration with others to name a few. 

We are driven to work hard and help others by seeing fellow community members do some awesome work. The thought "Collaboration 1st, Code 2nd" explains it all.

**Reply by kaiwalyakoparkar** [reply]
Awesome :tada: Glad you liked the community :smile:

**Reply by eddiejaoude** [reply]
Thank you @shikharcodes

**Comment by Cahllagerfeld** [comment]
Eddiehub is a place where everyone can benefit from everyone if they get involved.

**Reply by eddiejaoude** [reply]
Yes!! 💥  awesome 🤓

**Comment by Evavic44** [comment]
Eddiehub has helped me to understand the importance of Open Source and how easy it is to collaborate with developers from around the world to work on different projects and learn from each other in an amazing community.

**Reply by kaiwalyakoparkar** [reply]
That's true :100:

**Reply by eddiejaoude** [reply]
Wow thank you 🔥

**Comment by ayushtamra** [comment]
I joined Eddiehub during MLH Local hack day, and my experience with the community was really awesome.
Here everyone is treated equally and get a chance to explore and learn new things.
Not just that, everyone participate, organize, mentor different events like hackathon, conferences, public speaking.

**Reply by eddiejaoude** [reply]
thank you!! 🔥

**Comment by Bryson69** [comment]
EddieHub is the first open-source community I joined and since then my experience has been awesome. Contributing to open-source software and meeting new people who share a common goal. I also had the chance to make a great friend @vinzvinci . It's a community that has people who are willing to help you. It the best Open Source Community.

**Reply by eddiejaoude** [reply]
Great to hear 💥

**Comment by EmanuelGF** [comment]
I Started contributing and building my GitHub profile and already got a job interview request based on my GitHub activity. So I really appreciate what the EddieHub community is and does! I could not recommend enough joining the EddieHub community! (5/5)

**Reply by kaiwalyakoparkar** [reply]
Awesome :tada::heart::boom:

**Reply by eddiejaoude** [reply]
Amazing 🔥

**Comment by akhileshthite** [comment]
EddieHub is the best growing 🚀 Open-Source community. Here, you can help each other by raising issues, solving doubts on the help channel, show your work (projects), make good connections and so many things.
I would recommend all beginners to join the most respectful, collaborative and loving Open-Source community.
Thanks @eddiejaoude, we support you!

**Reply by eddiejaoude** [reply]
Thank you 👍

**Comment by mrunankpawar** [comment]
EddieHub has been life-changing for me. I still remember I joined EddieHub in January 2021 and I was very nervous during the meet being an introvert, but then everyone was supposed to introduce themselves and the community was so cheerful that I just went for it. I kept on attending such meets and now I'm confident while speaking to anyone. I also conduct sessions and that too confidently, so EddieHub FTW !!  (5/5)

**Reply by eddiejaoude** [reply]
Super great 💥

**Comment by suitupstan** [comment]
EddieHub is the place where my journey began, EddieHub tries to help everyone in every possible way they can! I didn't even know how to code when I joined, now am skilled enough to help other people. I love the gesture of the community, the community is built over strong pillars. Love how the community is really friendly, everyone is up for helping anyone. Eddie is awesome, the moderators are awesome and the community comprising all of this, plus really friendly members is even awesome 😄

**Reply by eddiejaoude** [reply]
Thank you so much Krish! Great to have you in the community 🔥

**Comment by Pradumnasaraf** [comment]
Joining **EddieHub** is one of the best decisions of my life. Everyone in the community is so encouraging, inspiring, and supportive, it feels like a family. By joining it, not only did I get a proper direction to contribute to Open Source and my career, but also it nurtured my soft skills and helped me a lot in personal development. Thank You **Eddie** & **Sara** for making such a beautiful community :)

⭐ 100/5 rating!

**Reply by eddiejaoude** [reply]
Thank you Pradumna for the amazing feedback! Great to have you in the commuity 💥

**Comment by Panquesito7** [comment]
EddieHub is one of the best and most inclusive communities I know. Here, everyone supports you, everyone helps you, and then you support other people, which gives a very good community feeling! 🎉 Everyone is awesome, all the moderators, maintainers, and community members are the best! ❤️ EddieHub has led me to achieve bigger things, as well as leading many communities and helping more people.

Thank you Eddie for making this awesome community! 🚀
EddieHub is invaluable: I can't rate it. :)

**Reply by eddiejaoude** [reply]
Thank you for the great feedback @Panquesito7 🔥

**Comment by Susmita-Dey** [comment]
**EddieHub** is the first community to help me start my journey to **Open Source**. I wasn't aware of anything called **"Open Source"** before I met @eddiejaoude on a Twitter space and came across **EddieHub community** and found so many people here geeking out, helping each other, making PRs and issues and many more. The repositories maintainers are so kind here; always ready to help each other. **I created my first issue and PR here** and contributed to this community and soon became a member. It helped me to customize my GitHub profile and taught me to ***collaborate with people all over the world.***
I'm very happy being here as a member now and helping people geeking out together and learning new things. ❤️

Thank you Eddie Sir for making this awesome community! 🚀
EddieHub is invaluable: It's beyond any ratings :)

**Reply by eddiejaoude** [reply]
Thank you @Susmita-Dey for the amazing feedback! 🔥
